### ISSUE TITLE
Checking if variable is datetime, when datetime is not imported 

### ISSUE THREAD

**Issue by dmulhall** [britned/empire-platform-api#5]
Using empire-platform-api-public-client==1.0.0

When calling TransmissionRightApi.get_transmission_rights_overview() I get the following error:

```python
 File "C:\Dev\powercapacitycapture\powercapacitycapture\empire\empire_capacity_capture.py", line 16, in get_empire_capacity
    auctions = auction_api.get_transmission_rights_overview(
  File "pydantic\decorator.py", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_function
    import itertools
  File "pydantic\decorator.py", line 134, in pydantic.decorator.ValidatedFunction.call
    if doc:
  File "pydantic\decorator.py", line 206, in pydantic.decorator.ValidatedFunction.execute
    name, rest = obj.strip().split('(', 1)
  File "C:\Dev\pyapps\powercapacitycapture\lib\site-packages\empire_platform_api_public_client\api\transmission_right_api.py", line 87, in get_transmission_rights_overview
    return self.get_transmission_rights_overview_with_http_info(delivery_day, x_participant_ids, border_direction, timescales, mtu_size, **kwargs)  # noqa: E501
  File "pydantic\decorator.py", line 40, in pydantic.decorator.validate_arguments.validate.wrapper_function
    import itertools
  File "pydantic\decorator.py", line 134, in pydantic.decorator.ValidatedFunction.call
    if doc:
  File "pydantic\decorator.py", line 206, in pydantic.decorator.ValidatedFunction.execute
    name, rest = obj.strip().split('(', 1)
  File "C:\Dev\pyapps\powercapacitycapture\lib\site-packages\empire_platform_api_public_client\api\transmission_right_api.py", line 174, in get_transmission_rights_overview_with_http_info
    if isinstance(_params['delivery_day'], datetime):
NameError: name 'datetime' is not defined
python-BaseException
``` 

This block of code looks to be the issue:

```python
if _params.get('delivery_day') is not None:  # noqa: E501
    if isinstance(_params['delivery_day'], datetime):
        _query_params.append(('deliveryDay', _params['delivery_day'].strftime(self.api_client.configuration.date_format)))
    else:
        _query_params.append(('deliveryDay', _params['delivery_day']))
```   

Fix would be to either import datetime or check if _params['delivery_day'] isinstance date

**Comment by exoszajzbuk** [comment]
@dmulhall thanks for pointing it out, we came aware of the datetime issue after following up on #4 

we'll be able to fix it in the coming 1.1.0 release

**Comment by exoszajzbuk** [comment]
@dmulhall The issue has been fixed with the 1.1.0 release of the SDK.

The Backend implementing the full 1.1.0 feature set of the API is only scheduled to be deployed on the TRAINING environment on 2023-11-08, but you can validate the fix using the 1.1.0 SDK

**Comment by exoszajzbuk** [comment]
@dmulhall  the 1.1.0 release of the platform (implementing API version 1.1.0) has been deployed on the TRAINING environment yesterday

can you please re-test your issue and confirm if it is fixed?
